cpu/stm32_common/stmclk.c

```diff
@@ -47,6 +47,16 @@
 #define PLL_SRC                  RCC_PLLCFGR_PLLSRC_HSI
 #endif
 
+#if defined(CPU_FAM_STM32F2)
+#define RCC_PLLCFGR_PLLP_Pos    (16U)
```

Comment:
Small nit: the defines could be aligned with the ones below

Reply:
Fixed and amended directly